packages/camera/camera_android_camerax/example/integration_test/integration_test.dart

```diff
@@ -19,6 +21,46 @@ void main() {
     CameraPlatform.instance = AndroidCameraCameraX();
   });
 
+  final Map<ResolutionPreset, Size> presetExpectedSizes =
+      <ResolutionPreset, Size>{
+    ResolutionPreset.low: const Size(240, 320),
+    ResolutionPreset.medium: const Size(480, 720),
+    ResolutionPreset.high: const Size(720, 1280),
+    ResolutionPreset.veryHigh: const Size(1080, 1920),
+    ResolutionPreset.ultraHigh: const Size(2160, 3840),
+    // Don't bother checking for max here since it could be anything.
+  };
+
+  /// Verify that [actual] has dimensions that are at least as large as
```

Comment:
For my understanding, why do we allow `actual` to have dimensions greater than the expected here? At first I thought it was for the case of fallback, but isn't the fallback rule being used here the `ResolutionStrategy.fallbackRuleClosestLower`, as set in the `...FromPreset` method? 
I would think that would mean we would be okay with `actual <= expected`, not `actual >= expected`.

Reply:
My understanding is that this method is testing `actual <= expected`.

Reply:
Oh, yes it looks like it is that way. In that case maybe change the comment to "at most as large as"